CONTRIBUTING.md

```diff
@@ -15,15 +15,15 @@ instructions provided by the bot. You will only need to do this once across all
 
 ## Linux
 
-You'll need [bazel](https://docs.bazel.build/versions/master/install-ubuntu.html) and g++-10 to use project from source:
+You'll need [bazel](https://docs.bazel.build/versions/master/install-ubuntu.html) and g++-11 to use project from source:
 
 ```bash
 sudo apt install curl gnupg
 curl -fsSL https://bazel.build/bazel-release.pub.gpg | gpg --dearmor > bazel.gpg
 sudo mv bazel.gpg /etc/apt/trusted.gpg.d/
 echo "deb [arch=amd64] https://storage.googleapis.com/bazel-apt stable jdk1.8" | sudo tee /etc/apt/sources.list.d/bazel.list
 
-sudo apt update && sudo apt install bazel clang-format g++-10
+sudo apt update && sudo apt install bazel clang-format g++-11
```

Comment:
I don't think G++11 is supported on ubuntu 20.

Reply:
We are not using it though. Our default compiler is [g++-11](https://github.com/microsoft/DeepGNN/blob/main/.bazelrc#L6)